grow/commands/build.py

```diff
@@ -16,7 +16,8 @@
 @click.option('--clear_cache',
               default=False, is_flag=True,
               help='Clear the pod cache before building.')
-def build(pod_path, out_dir, preprocess, clear_cache):
+@click.option('--files', help='Build only pages affected by content files.', multiple=True)
```

Comment:
I like `pod_path` because that's super clear (doesn't conflict with "path" which might mean URL path). It's just a bit verbose. I'm trying to see if click has a way to use an option with one name but the kwarg to the function be named something else...